service/src/main/java/uk/gov/hmcts/reform/fpl/service/MessageJudgeService.java

```diff
@@ -106,4 +212,8 @@ private boolean hasSelectedC2(CaseData caseData) {
         return hasC2Documents(caseData)
             && caseData.getMessageJudgeEventData().getC2DynamicList() != null;
     }
+
+    private boolean hasJudicialMessages(CaseData caseData) {
+        return caseData.getJudicialMessages() != null && !caseData.getJudicialMessages().isEmpty();
```

Comment:
`caseData.getJudicialMessages() != null` will always be true as the method returns an empty list if null